as_perimenoun_oi_gynaikes.ass

```diff
@@ -556,47 +557,47 @@ Dialogue: 0,0:41:08.99,0:41:10.76,Default,,0,0,0,,Is it the refs fault \Nif you
 Dialogue: 0,0:41:10.76,0:41:13.56,Default,,0,0,0,,- Was there a score reversal at the last minute?\N- Three meters from the penalty box?!
 Dialogue: 0,0:41:13.56,0:41:15.99,Default,,0,0,0,,Are you kidding me?\N Manolas tackles Skartados...
 Dialogue: 0,0:41:15.99,0:41:18.64,Default,,0,0,0,,...sends him airborne for three meters...
-Dialogue: 0,0:41:18.64,0:41:21.09,Default,,0,0,0,,...and falls in the penatly box!\N - He was faking it!
+Dialogue: 0,0:41:18.64,0:41:21.09,Default,,0,0,0,,...and falls in the penatly box!\N - Was he a bird?!
 Dialogue: 0,0:41:21.09,0:41:24.84,Default,,0,0,0,,And the boy is lying half-dead in the mud,\Nand the refs shows him the yellow card...
-Dialogue: 0,0:41:24.84,0:41:27.09,Default,,0,0,0,,...as if he is faking it? And he still has a heartbeat!?
+Dialogue: 0,0:41:24.84,0:41:27.09,Default,,0,0,0,,...as if he is faking it? And he is still lives!?
 Dialogue: 0,0:41:27.09,0:41:30.84,Default,,0,0,0,,As if we were waiting for the Laconian farmers \Nto come and teach us what a penalty is...
 Dialogue: 0,0:41:30.84,0:41:35.04,Default,,0,0,0,,- Was it a penalty Mr. Panos?\N- Miss, it was the {\i1}definition{\i0} of a penalty!
 Dialogue: 0,0:41:35.04,0:41:38.01,Default,,0,0,0,,But Thessaloniki must install a customs office!
 Dialogue: 0,0:41:38.01,0:41:42.09,Default,,0,0,0,,Where was your passport issued mister? \NIn Laconia? No entry!
 Dialogue: 0,0:41:42.09,0:41:43.84,Default,,0,0,0,,No entry!
 Dialogue: 0,0:41:44.51,0:41:46.26,Default,,0,0,0,,No entry!
 Dialogue: 0,0:41:46.76,0:41:51.21,Default,,0,0,0,,Antonis, I am afraid. Since the doctors said that \NMyrto must be committed to the psychiatric hospital...
-Dialogue: 0,0:41:51.21,0:41:53.36,Default,,0,0,0,,...he sits still in the smith shop for hours on end.
-Dialogue: 0,0:41:53.36,0:41:55.36,Default,,0,0,0,,What happened with the girl?
-Dialogue: 0,0:41:55.36,0:41:58.06,Default,,0,0,0,,We passed his car as we were supposed \Nto and he gets out of the Mercedes.
+Dialogue: 0,0:41:51.21,0:41:53.36,Default,,0,0,0,,...he sits still in the smith shop for hours.
+Dialogue: 0,0:41:53.36,0:41:55.36,Default,,0,0,0,,What exactly happened with the girl?

```

Comment:
Not an improvement to the translation (we are not going for 100% fidelity, less is more when reading subtitles and this move is already crowded).